tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathHeader.txt

```diff
@@ -285,26 +285,15 @@
-//ldml/personNames/personName[@length=\"%A\"][@usage=\"%A\"]/namePattern[@_q=\"%A\"]                ; Misc ; Person Name Formats ; Name Patterns for Length $1 ; usage-$2-style-any-order-any
-//ldml/personNames/personName[@length=\"%A\"][@style=\"%A\"][@order=\"%A\"]/namePattern[@_q=\"%A\"]         ; Misc ; Person Name Formats ; Name Patterns for Length $1 ; usage-any-style-$2-order-$3
-//ldml/personNames/personName[@length=\"%A\"][@style=\"%A\"]/namePattern[@_q=\"%A\"]                ; Misc ; Person Name Formats ; Name Patterns for Length $1 ; usage-any-style-$2-order-any
-//ldml/personNames/personName[@length=\"%A\"][@order=\"%A\"]/namePattern[@_q=\"%A\"]                ; Misc ; Person Name Formats ; Name Patterns for Length $1 ; usage-any-style-any-order-$2
-//ldml/personNames/personName[@length=\"%A\"]/namePattern[@_q=\"%A\"]                       ; Misc ; Person Name Formats ; Name Patterns for Length $1 ; usage-any-style-any-order-any
-
-//ldml/personNames/personName[@usage=\"%A\"][@style=\"%A\"][@order=\"%A\"]/namePattern[@_q=\"%A\"]         ; Misc ; Person Name Formats ; Name Patterns for Length any ; usage-$1-style-$2-order-$3
-//ldml/personNames/personName[@usage=\"%A\"][@style=\"%A\"]/namePattern[@_q=\"%A\"]                 ; Misc ; Person Name Formats ; Name Patterns for Length any ; usage-$1-style-$2-order-any
-//ldml/personNames/personName[@usage=\"%A\"][@order=\"%A\"]/namePattern[@_q=\"%A\"]                 ; Misc ; Person Name Formats ; Name Patterns for Length any ; usage-$1-style-any-order-$2
-//ldml/personNames/personName[@usage=\"%A\"]/namePattern[@_q=\"%A\"]                        ; Misc ; Person Name Formats ; Name Patterns for Length any ; usage-$1-style-any-order-any
-//ldml/personNames/personName[@style=\"%A\"][@order=\"%A\"]/namePattern[@_q=\"%A\"]                 ; Misc ; Person Name Formats ; Name Patterns for Length any ; usage-any-style-$1-order-$2
-//ldml/personNames/personName[@style=\"%A\"]/namePattern[@_q=\"%A\"]                        ; Misc ; Person Name Formats ; Name Patterns for Length any ; usage-any-style-$1-order-any
-//ldml/personNames/personName[@order=\"%A\"]/namePattern[@_q=\"%A\"]                        ; Misc ; Person Name Formats ; Name Patterns for Length any ; usage-any-style-any-order-$1
-
+//ldml/personNames/nameOrder[@nameLocales=\"%A\"]   ; Misc ; Person Name Formats ; Name Order For Languages ; $1
+//ldml/personNames/personName/namePattern[@alt=\"%A\"]      ; Misc ; Person Name Formats ; Fallback Name Pattern ; any-$1
+//ldml/personNames/personName/namePattern           ; Misc ; Person Name Formats ; Fallback Name Pattern ; any
+//ldml/personNames/personName[@length=\"%A\"][@usage=\"%A\"][@style=\"%A\"][@order=\"%A\"]/namePattern[@alt=\"%A\"]     ; Misc ; Person Name Formats ; Name Patterns for Length $1 ; $1-$2-$3-$4-$5
```

Comment:
For the ST, the same as above; the full attributes are enough.
However, for the charts, I think we need all of them.
Double however, because the 'code' will always be just all of the attributes concatenated, I think we can use something like:

... [@%att=\"%A\"][@%att=\"%A\"][@%att=\"%A\"][@%att=\"%A\"]/namePattern[@alt=\"%A\"]
... [@%att=\"%A\"][@%att=\"%A\"][@%att=\"%A\"][@%att=\"%A\"]/namePattern
... [@%att=\"%A\"][@%att=\"%A\"][@%att=\"%A\"]/namePattern[@alt=\"%A\"]
...

Reply:
See my comment about coverage. I think that the regular data should *not* have entries for all values of ignored attributes, it should should just have entries without that attribute (i.e. for monograms no entries with usage attribute). So I have set up PathHeader, coverage etc. with that assumption. That is different than what we discussed in the meeting; I mentioned that in the PR overview and also in an e-mail.

Reply:
Ok. I still think we will have to do something for the charts, because they will need to work with production data. For them I think we will need to have a (simple) function &hyphen(...) for the output, that replaces spaces with hyphens. That way for the compacted data
length="long short" => long-short

However, we don't need to do that here.